File: 01.JAVA/pg-166-169/pg_167_16.java
import java.util.Scanner;

public class pg_167_16
{
    public static void main(String []args)
    {
        Scanner enter = new Scanner(System.in);
        int num;
        int dig1 = 0;
        int dig2 = 0;
        int dig3 = 0;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ A THREE-DIGITS INTEGER AND DETERMINE IF AT LEAST TWO OF THREE DIGITS ARE THE SAME");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");

        System.out.print("Write a three-digits integer: ");

        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }

        if(num > 100 && num < 999)
        {
            dig3 = num % 10;
            dig2 = (num / 10) % 10;
            dig1 = (num / 100) % 10;
        }
        else
        {
            System.out.println("The written number doesn't have three digits. Please try again!");
        }

        System.out.println(dig1 + "-" + dig2 + "-"+ dig3);

        if(dig1 == dig2)
        {
            System.out.println("The first and second digit are the same.");
        }
        else if(dig1 == dig3)
        {
            System.out.println("The first and thrid digit are the same.");
        }
        else if(dig2 == dig3)
        {
            System.out.println("The second and third digit are the same.");
        }
        else
        {
            System.out.println("None of the digits are the same.");
        }


    }
}

File: 03.JAVASCRIPT/pg-166-169/pg_167_16.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ A THREE-DIGITS INTEGER AND DETERMINE IF AT LEAST TWO OF THE THREE DIGITS ARE THE SAME");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput = document.querySelector(".num");
let btnProof = document.querySelector("#btnProof");
let textar = document.querySelector(".textar");

let dig1;
let dig2;
let dig3;

function validateNumber()
{
    let num = Number(numInput.value);

    if(num < 0)
    {
        num *= -1;
    }

    if(num >= 100 && num <= 999)
    {
        dig3 = num % 10;
        dig2 = Math.floor(num/10) % 10;
        dig1 = Math.floor(num/100) % 10;
    }
    else
        {
            textar.value = "The written number doesn't have three digits. Please try again!";
            return;
        }

    if(dig1 == dig2 && dig1 == dig3)
    {
        textar.value = "All digits are the same!";
    }
    else if(dig1 == dig2)
    {
        textar.value = "The first and second digit are the same.";
    }
    else if(dig1 == dig3)
    {
        textar.value = "The first and third digit are the same.";
    }
    else if(dig2 == dig3)
    {
        textar.value = "The second and third digit are the same.";
    }
    else
    {
        textar.value = "None digit are the same.";
    }
}

btnProof.addEventListener("click", validateNumber);
numInput.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});